docs/source/usage/configuration.rst

```diff
@@ -193,25 +193,35 @@ Computational benchmark post-processing configuration
 It is possible to control (to some extent) the post-processing of each benchmark via its 
 specific configuration file. These files are located in the ``<JADE_root>\Configuration\Benchmarks_Configuration``
 folder and their name must be identical to the one used in the ``File Name`` field in the main configuration file
-(using the .xlsx extension instead of the .i). These files are available only for computational benchmarks,
+(using the .yaml extension instead of the .i). These files are available only for computational benchmarks,
 since the high degree of customization needed for an experimental benchmark makes quite difficult to 
 standardize them. While computational benchmarks can be added to the JADE suite without the need for additional
 coding, this is not true also for experimental one.
 
-The files contain two main sheets, that respectively regulate the Excel and the Word/PDF (i.e., Atlas) post-processing output.
+The files contain two main sections, that respectively regulate the Excel and the Word/PDF (i.e., Atlas) post-processing output.
 
 Excel
 -----
 
-.. image:: ../img/conf/excelbench.png
-    :width: 600
+.. codeblock:: yaml
+
+    # Example of an extract from a benchmark configuration file, Excel section
+    Excel:
+        4.0:
```

Comment:
Some cases the tally number is a float, other cases just an integer. Minor, I guess just by product of conversion script?